qemu/tests/image_create_with_large_size.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest.qemu_storage import QemuImg
+
+
+def run(test, params, env):
+    """
+    Creating image with large size.
+    1. Create a qcow2 image with large size and check the output info.
+    2. Create a qcow2 with a normal size.
+    3. Increase a large size to the qcow2 image and check the output info.
+    :param test: Qemu test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    large_img = params["images"].splite()[1]
+    middle_img = params["images"].splite()[-1]
+    root_dir = data_dir.get_data_dir()
+    large = QemuImg(params.object_params(large_img), root_dir, large_img)
+    middle = QemuImg(params.object_params(middle_img), root_dir, middle_img)
+    large_filename = large.image_filename
+    size_increases = params["size_changes"]
+    create_err_info = "The image size is too large for file format " \
+                      "'qcow2' (try using a larger cluster size)"
+    resize_err_info = "Failed to grow the L1 table: File too large"
+
```

Comment:
better to define a function to wrap the `try` block.

Reply:
For `resize` function change to assert the output not `exception`, there seems no need to define a function. Check again, please.